constellations/gemini-admin/src/help/help.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+require("./help.styl");
+import React from "react";
+
+export class HelpPage extends React.Component {
+    render() {
+        return (
+            <div>
```

Comment:
it's not necessary but you could possibly create a title for the page called like help or like administrative support?